### ISSUE TITLE
localhost: breakage

### ISSUE THREAD

**Issue by pmitros** [uBlockOrigin/uBlock-issues#1902]
### Prerequisites

- [X] I read and understood the [policy as to what is a valid filter issue](https://github.com/uBlockOrigin/uAssets/blob/master/README.md#uassets)
- [X] I verified that the issue has not already been reported (use [this button](https://user-images.githubusercontent.com/585534/146582579-c32707a0-36a1-4cc5-ad50-83172c9f67a8.png) to find out)
- [X] I forced an update of my filter lists (**how to do this:** [click _"Purge all caches"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#purge-all-caches), then [click _"Update now"_](https://github.com/gorhill/uBlock/wiki/Dashboard:-Filter-lists#update-now))
- [X] I did not remove any of the [default filter lists](https://user-images.githubusercontent.com/585534/91296476-ab9d8c00-e76a-11ea-9fb9-87d6861bdf73.png)
- [X] I have no custom filter lists, or I verified that the issue still occurs without custom filter lists
- [X] I am not using [another content blocker along uBO](https://twitter.com/gorhill/status/1033706103782170625)

### URL address of the web page

`http://localhost:8888/`

### Category

breakage

### Describe the issue

uBlock Origin will block: 

/static/3rd_party/d3.v5.min.js
/static/3rd_party/require.js

Both on localhost:8888. This is the safest thing in the world. These are standard libraries, and not using a CDN preserves privacy. 

I'm pretty sure these weren't blocked until recently. 

### Screenshot(s)

<details>
<summary>Screenshot(s)</summary>

</details>

### Configuration

<details>

```yaml
uBlock Origin: 1.40.0
Firefox: 95
filterset (summary): 
  network: 78235
  cosmetic: 40066
  scriptlet: 16375
  html: 608
listset (total-discarded, last updated): 
  default: 
    ublock-filters: 30597-30, 1d.22h.45m
    ublock-badware: 3513-1, 1d.22h.43m
    ublock-privacy: 180-0, 1d.22h.41m
    ublock-abuse: 81-0, 1d.22h.39m
    ublock-unbreak: 1712-0, 1d.22h.37m
    easylist: 62197-565, 1d.22h.35m
    easyprivacy: 26543-475, 1d.22h.33m
    urlhaus-1: 7994-0, 11h.37m
    plowe-0: 3687-3, 12d.2h.17m
filterset (user): [empty]
modifiedUserSettings: [none]
modifiedHiddenSettings: [none]
supportStats: 
  launchToReadiness: 278 ms (selfie)
popupPanel: 
  blocked: 2
```
</details>

**Comment by gorhill** [comment]
I was able to reproduce using only these two filters and all other filter lists disabled:

    /^/$script,domain=example.*|bar.*
    .js|$script,domain=foo.*|bar.*

The two filters were needed to reproduce, I wasn't able to reproduce with only one or the other.

**Comment by gorhill** [comment]
Until the fix is publish in 1.40.2, a workaround which should work is to visit another site, for example <https://example.org>`, then reload the `localhost` page.

**Comment by pmitros** [comment]
Thank you for the super-quick fix! That was impressive.
